src/pages/dashboard.tsx

```diff
@@ -0,0 +1,7 @@
+export default function Dashboard({ isAuthorized, userID, todos }) {
+  return (
+    <div>
+      <p>You've logged in!</p>
```

Comment:
Vercel failed deployment because of the apostrophe. I think we can fix that by just changing this to 

```
<p>{`You've logged in!`}</p>
```